### PR TITLE
Add example of setting Content-Type header

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/6808

### CODE REVIEW

File: files/en-us/learn/server-side/configuring_server_mime_types/index.html [hunk 1]
```diff
@@ -109,14 +109,20 @@ <h2 id="How_to_set_up_your_server_to_send_the_correct_MIME_types">How to set up
...
- <li>If you're using NGINX, look at the <a href="/en-US/NGINX_configuration_snippets">NGINX configuration snippets</a>. NGINX does not have a <code>.htaccess</code> equivalent tool, so all changes will go into the main configuration file.</li>
+ <li>If you're using NGINX, note that NGINX does not have a <code>.htaccess</code> equivalent tool, so all changes will go into the main configuration file.</li>
  <li>If you're using a server-side script or framework to generate content, the way to indicate the content type will depend on the tool you're using. Check the framework or library's documentation.</li>
 </ul>
 
+<p>Regardless of what server system you use, the effect you need to achieve is to set a response header with the name {{httpheader("Content-Type")}}, followed by a colon and space, followed by a MIME type. For example, in a PHP environment, you’d need to set the response header for PDF resources like this:
```
Comment (teoli2003) [inline files/en-us/learn/server-side/configuring_server_mime_types/index.html:116]: I think this is not 100% accurate. You can set it up at the server level as described above, or you can set it up at the php level with `header()`. The sentence sounds like you need to do it at the php level, in addition to the web server level
```suggestion
<p>Regardless of what server system you use, the effect you need to achieve is to set a response header with the name {{httpheader("Content-Type")}}, followed by a colon and space, followed by a MIME type. High level environments often allow such headers to be set when generating the page. For example, in a PHP environment, you could set the response header for PDF resources like this:
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/learn/server-side/configuring_server_mime_types/index.html [hunk 2]
```diff
@@ -109,14 +109,20 @@ <h2 id="How_to_set_up_your_server_to_send_the_correct_MIME_types">How to set up
...
- <li>If you're using NGINX, look at the <a href="/en-US/NGINX_configuration_snippets">NGINX configuration snippets</a>. NGINX does not have a <code>.htaccess</code> equivalent tool, so all changes will go into the main configuration file.</li>
+ <li>If you're using NGINX, note that NGINX does not have a <code>.htaccess</code> equivalent tool, so all changes will go into the main configuration file.</li>
  <li>If you're using a server-side script or framework to generate content, the way to indicate the content type will depend on the tool you're using. Check the framework or library's documentation.</li>
 </ul>
 
+<p>Regardless of what server system you use, the effect you need to achieve is to set a response header with the name {{httpheader("Content-Type")}}, followed by a colon and space, followed by a MIME type. High level environments often allow such headers to be set when generating the page. For example, in a PHP environment, you could set the response header for PDF resources like this:
```
Comment (sideshowbarker) [inline files/en-us/learn/server-side/configuring_server_mime_types/index.html:116]:
```suggestion
<p>Regardless of what server system you use, the effect you need to achieve is to set a response header with the name {{httpheader("Content-Type")}}, followed by a colon and space, followed by a MIME type. High-level environments often allow such headers to be set when generating the page. For example, in a PHP environment, you could set the response header for PDF resources like this:
```

**Review by teoli2003 — APPROVED** [review]

**Comment by David263** [comment]
Thank you for this change. I've not been able as yet to understand how GitHub works. Can you provide instructions for viewing the changed lines, please?

**Comment by sideshowbarker** [comment]
> Thank you for this change. I've not been able as yet to understand how GitHub works. Can you provide instructions for viewing the changed lines, please?

* https://github.com/mdn/content/pull/6881/files has the diff for the change
* https://pr6881.content.dev.mdn.mozit.cloud/en-US/docs/Learn/Server-side/Configuring_server_MIME_types#how_to_set_up_your_server_to_send_the_correct_mime_types has a preview of the rendered version of the article, with the change incorporated

**Pull request merged**